### ISSUE TITLE
https://help.geo.admin.ch/?ids=54#/?ids=62&lang=de does not fit 

### ISSUE THREAD

**Issue by davidoesch** [geoadmin/web-geoadmin-help#15]
https://help.geo.admin.ch/?ids=62&lang=de

![image](https://user-images.githubusercontent.com/4577727/75880358-66cd1400-5e1d-11ea-911e-f044159e2ab6.png)

**Comment by davidoesch** [comment]
seems to be solved now ---- did you fix it @pakb ?

**Comment by pakb** [comment]
nope, can still reproduce it if you squeeze a bit the viewport with dev tools. but the overflow is minimal (barely visible if you don't know it, text is touching the side which should not happen as we have a padding)
